Ikuyadeu/vscode-R
# WIP Migrate to vscode-test #315

Closes #315 

**What problem did you solve?**

Unit tests were not working. This PR migrates unit tests to `vscode-test`. Now unit tests work again.

**How can I check this pull request?**

1. Run -> Start Debugging...
1. Select 'Launch Tests' from the dropdown
1. Press <kbd>F5</kbd>
1. Observe Debug Console shows output ending with

```
    ✓ Selecting multi-line bracket with pipe    spec.js:76
    ✓ Selecting shorter RStudio comparison      spec.js:76
  22 passing (164ms)
```
